skypy/galaxy/tests/test_luminosity.py

```diff
@@ -36,12 +36,10 @@ def calc_pdf(luminosity, z, alpha, a_m, b_m, absolute_magnitude_max=-16.0,
         luminosity_max = astro.luminosity_from_absolute_magnitude(
             absolute_magnitude_min)
         lum_star = lum._calculate_luminosity_star(z, a_m, b_m)
-        c = special.upper_incomplete_gamma(alpha + 1,
-                                           luminosity_min / lum_star)
-        d = special.upper_incomplete_gamma(alpha + 1,
-                                           luminosity_max / lum_star)
-        return 1. / lum_star * np.power(luminosity / lum_star, alpha) * np.exp(
-            - luminosity / lum_star) / (c - d)
+        c = special.gammaincc(alpha+1, luminosity_min/lum_star)
+        d = special.gammaincc(alpha+1, luminosity_max/lum_star)
+        return 1./lum_star*np.power(luminosity/lum_star, alpha)*np.exp(
+                -luminosity/lum_star)/(c - d)
```

Comment:
I suggest that we do normalise the pdf properly; somebody in the future might change the cdf integration method without realising the pdf is not normalised.
```suggestion
        from scipy.special import gamma
        c = special.gammainc(alpha+1, luminosity_min/lum_star) / gamma(alpha+1)
        d = special.gammainc(alpha+1, luminosity_max/lum_star) / gamma(alpha+1)
        return 1./lum_star*np.power(luminosity/lum_star, alpha)*np.exp(
                -luminosity/lum_star)/(c - d)
```

Reply:
That whole test is actually broken because the default upper and lower limits of the luminosity sampling do not coincide with the upper and lower limits given to the CDF. The explicit normalisation just papered that over. Will need a separate PR to fix.

Reply:
Oh ja I just checked it. Apparently, I forgot to change the test when we decided to use the units of L_*.